packages/quantic/force-app/main/default/lwc/quanticRecentResultsList/quanticRecentResultsList.js

```diff
@@ -102,6 +104,7 @@ export default class QuanticRecentResultsList extends LightningElement {
       }
     });
     this.unsubscribe = this.recentResultsList.subscribe(() => this.updateState());
+    this.showPlaceholder = false;
```

Comment:
Same as previous comment. I think `this.showPlaceholder = false` should be moved into `updateState()`.